dist/colors.android.xml

```diff
@@ -0,0 +1,61 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<resources>
+  <property name="COLOR-PURPLE-LIGHTER" category="background-color" value="#fff6f0fd"/>
```

Comment:
Hi @kaelig please find a sample of how the resources are formatted for usage on android:
` <color name="colorPrimary">@color/polaris_titlebar_dark</color>`

Adding @see-mack  who is working on the polaris components for Android.

Reply:
More specifically, a proper format with a hex value would be something like `<color name="polaris_indigo">#5c6ac4</color>`

This is the resource file we currently use: https://github.com/Shopify/android/blob/master/Polaris/Core/src/main/res/values/colors.xml

Reply:
@see-mack I updated the PR, can you please have another look?